data/org.mupen64plus.Mupen64Plus.appdata.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>org.mupen64plus.Mupen64Plus</id>
+  <name>Mupen64Plus</name>
+  <summary>Nintendo 64 Emulator</summary>
+  <metadata_license>CC-BY-SA-3.0</metadata_license>
+  <project_license>GPL-2.0+</project_license>
+  <description>
+    <p>
+      Mupen64plus is a Nintendo 64 Emulator.
+      This package includes all the plug-ins.
+    </p>
+  </description>
+  <url type="homepage">http://mupen64plus.org/</url>
+  <url type="help">http://mupen64plus.org/docs/</url>
+  <url type="bugtracker">https://github.com/mupen64plus/mupen64plus-user-issues/issues</url>
+  <url type="donation">https://www.paypal.com/webapps/shoppingcart?flowlogging_id=59a5b20d47d74&mfid=1541895691620_59a5b20d47d74</url>
+  <screenshots>
+    <screenshot type="default">
+      <image>https://screenshots.debian.net/screenshots/000/010/356/large.png</image>
+      <caption>Command line interface help out</caption>
+    </screenshot>
+  </screenshots>
+  <releases>
+    <release date="2015-04-26" version="2.5"/>
```

Comment:
Is upstream still developed?

Reply:
Ah, good catch

Reply:
Yes, but it's been a very long time since they've had a release.

(It looks like they broke up their source into a bunch of different modules in going to git hub, and have updated some of the modules, but not the "core")